migrations/2024-01-30-090815_alter_payout_type/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
+SELECT 1;
```

Comment:
Is this supposed to be this?

Reply:
Yes, it is the default case since we don't necessarily need to rollback for a type alter👍